packages/animations/browser/src/dsl/animation_timeline_builder.ts

```diff
@@ -586,7 +586,6 @@ export class AnimationTimelineContext {
 
 export class TimelineBuilder {
   public duration: number = 0;
-  // TODO(issue/24571): remove '!'.
   public easing !: string | null;
```

Comment:
To remove the ! we need to initialize this property. In this case we can use null.

```suggestion
  public easing: string | null = null;
```

Reply:
@petebacondarwin how would we do the others such as this?

```
  public parentPlayer !: AnimationPlayer;
```

or this?

```
public readonly domPlayer !: DOMAnimation;
```

If I can get it, I'm happy to pair with @Menuka5 if he's still up for it. :)

Reply:
The instructions for deciding if the `!:` can be removed are provided in https://github.com/angular/angular/issues/24571: 

For angular components, use the following rules in deciding between:
a) adding initializer
b) make the field optional
c) leave the '!'

- If the field is annotated with @Input - Make the field optional b) or add an initializer a).
- If the input is required for the component user - add an assertion in ngOnInit and apply c.
- If the field is annotated @ViewChild, @ContentChild - Make the field optional b).
- If the field is annotated with @ViewChildren or @ContentChildren - Add back '!' - c).
- Fields that have an initializer, but it lives in ngOnInit. - Move the initializer to the constructor.
- Fields that have an initializer, but it lives in ngOnInit and cannot be moved because it depends on other @Input fields - Add back '!' - c).

Reply:
I like to help 😃

Reply:
Hi all, 

May I break this change into a few pull requests? Because this will affect 151 lines of change and going through all of might be hard for a reviewer.

Reply:
Absolutely. I would start with simply one or two cases so that you can get good feedback and confidence in the process. Then if you want to do more, create separate PRs for each "area" - e.g. core, compiler, forms, etc.

Reply:
let me know if I could be of any help here...I am into Angular 7 front-end development but a newbie to open source world...Happy to jump in!

Reply:
@Menuka5, and @dkbollaram I'm around too, if either of you need help on getting the PRs through the review process. I'm also happy to collaborate, but it's definitely new territory for me.